mixes/reducer.js

```diff
@@ -29,10 +34,12 @@ function createReducer (config) {
       Effects.promise(runLoadMixList),
       Effects.constant(loadMixListEnd())
     ])),
-    [loadMixListSuccess]: (state, action) => ({
-      ...state,
-      records: merge({}, state.records, keyBy(action.payload, 'id'))
-    }),
+
+    // how to get this to not run LOAD_MIX_END end until all SET_MIX are done?
```

Comment:
yeah i think so. here's documentation on how `Effects.batch` works: https://github.com/redux-loop/redux-loop/blob/3.0/docs/ApiDocs.md#effectsbatcheffects, but in this case it might be dispatching the end action _after the promise resolves_ but in this case is before everything is set? an alternative approach could be to manually loop the end action from within the success and failure actions.